indexer/altitude_loader.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "indexer/feature_altitude.hpp"
+#include "indexer/index.hpp"
+
+#include "3party/succinct/rs_bit_vector.hpp"
+
+#include "std/unique_ptr.hpp"
+#include "std/vector.hpp"
+
+namespace feature
+{
+using TAltitudeSectionVersion = uint16_t;
+using TAltitudeSectionOffset = uint32_t;
+
+class AltitudeLoader
+{
+public:
+  explicit AltitudeLoader(MwmValue const * mwmValue);
+
+  TAltitudes GetAltitude(uint32_t featureId, size_t pointCount) const;
+
+private:
+  void DeserializeHeader(ReaderSource<FilesContainerR::TReader> & rs);
+
+  vector<char> m_altitudeAvailabilitBuf;
```

Comment:
MemoryRegion and Co are defined in indexer/rank_table.cpp. It would nice to move the definitions to indexer/rank_table.hpp.